src/Hedgehog/Property.fs

```diff
@@ -137,21 +140,25 @@ module private Pretty =
             (renderTests tests)
 
     let renderFailed
+            (size : Size)
+            (seed : Seed)
             (tests : int<tests>)
             (discards : int<discards>)
             (shrinks : int<shrinks>)
             (journal : Journal) : string =
         let sb = StringBuilder ()
 
-        sprintf "*** Failed! Falsifiable (after %s%s%s):"
+        renderf sb "*** Failed! Falsifiable (after %s%s%s):"
             (renderTests tests)
             (renderAndShrinks shrinks)
             (renderAndDiscards discards)
-            |> append sb
 
         List.iter (append sb) (Journal.toList journal)
 
-        sb.ToString(0, sb.Length - 1) // exclude extra newline
+        renderf sb "This failure can be reproduced by running:"
+        renderf sb "> Property.recheck (%d : Size) (%0A) <property>" size seed
```

Comment:
(Repeating my suggestion from https://github.com/hedgehogqa/fsharp-hedgehog/pull/233#issuecomment-735229874)

I think it would be better if this were only printed when `Property.check` is called and not also printing it when `Property.recheck` is called.

Reply:
I don't think it's a huge blocker if everything else works fine. If it's something easy to do, let's do it... If not, we can wrap up, merge and do a release. (And get back to this in a separate PR.)

Reply:
I agree with that.  I am willing to improve this at a later time.

With that bug fix in https://github.com/hedgehogqa/fsharp-hedgehog/pull/233/commits/a697cc135f6092fa2c8f2107767dedbb715fba62, the branch in this PR is now a great improvement over `master`.

Reply:
I did manage to make this change. In doing so, I reduced the arity of some of the functions in the call stack of `recheck`.

Reply:
Indeed you did.  Excellent work! :D